tests/test_workers/test_tasks/test_build.py

```diff
@@ -1004,18 +1004,80 @@ def test_handle_rm_request(
 
 
 @mock.patch('iib.workers.tasks.build._cleanup')
-@mock.patch('iib.workers.tasks.build.run_cmd')
+@mock.patch('iib.workers.tasks.build.prepare_request_for_build')
+@mock.patch('iib.workers.tasks.build._update_index_image_build_state')
+@mock.patch('iib.workers.tasks.build.opm_registry_rm_fbc')
```

Comment:
isn't this function in opm_operations?

Reply:
I had to patch the where the function is called as well